Source/src/WixSharp/CommonTasks.cs

```diff
@@ -503,6 +503,29 @@ static public Project AddDirs(this Project project, params Dir[] items)
             return project;
         }
 
+        /// <summary>
+        /// Adds the IniFile to the Project.
+        /// </summary>
+        /// <param name="project">The project.</param>
+        /// <param name="item">The item.</param>
+        /// <returns></returns>
+        static public Project AddIniFile(this Project project, Dir item)
```

Comment:
I was mistaken here.
Should be:
`
static public Project AddIniFile(this Project project, IniFile item)
`